### PR TITLE
Implement fallback to deviceinfo if getprop fails

### LINKED ISSUE #9: Provide alternative device type determination for Non-Android devices
Currently system-image calls getprop to determine the device type. This is not supported on Non-Android phones, and needs a replacement.

### PR DESCRIPTION
**Pull request by Flohack74**
This is needed for Non-Android phones that do not have getprop implemented. Should fix #9

### CODE REVIEW

File: systemimage/device.py [hunk 1]
```diff
@@ -45,6 +45,11 @@ def get_device(self):
...
         except FileNotFoundError as error:
-            log.exception('getprop command not found')
-            return '?'
+            # Try to use device-info instead
+            log.exception('getprop command not found, trying fallback to deviceinfo')
+            try:
+                stdout = check_output(
+                    'device-info get Name'.split(), universal_newlines=True)
+            except:
+                return '?'
```
Comment (UniversalSuperBox) [inline systemimage/device.py]: You must specify an exception to catch in Python. This is a syntax error.

**Review by UniversalSuperBox — CHANGES_REQUESTED** [review]

File: systemimage/device.py (hunk 1 above)
Comment (Flohack74) [reply systemimage/device.py]: I dont think so, this should match all exceptions?

Comment (Flohack74) [reply systemimage/device.py]: See https://stackoverflow.com/questions/4990718/about-catching-any-exception

**Comment by NeoTheThird** [comment]
Is this still needed?

**Comment by Flohack74** [comment]
Yes but does not work still and I have no Pinephone to debug it.

**Comment by Flohack74** [comment]
I close heree in favour of #11

**Pull request closed without merging**